components/changelog/ChangelogTrigger.js

```diff
@@ -55,17 +57,29 @@ const ChangelogTrigger = props => {
     });
   };
 
+  const TooltipContent = (
+    <FormattedMessage id="ChangelogTrigger.tooltip.content" defaultMessage="What's new with Open Collective" />
+  );
+
   if (!LoggedInUser || !CHANGE_LOG_UPDATES_ENABLED) {
     return null;
   }
 
   return (
     <Flex>
       {hasSeenNewUpdates ? (
-        <FlameIcon onClick={handleShowNewUpdates} backgroundColor="black.100" url="/static/images/flame-default.svg" />
+        <StyledTooltip content={TooltipContent}>
+          <FlameIcon
+            onClick={handleShowNewUpdates}
+            backgroundColor="black.100"
+            url="/static/images/flame-default.svg"
+          />
+        </StyledTooltip>
       ) : (
         <Dropdown>
-          <FlameIcon onClick={handleShowNewUpdates} backgroundColor="yellow.100" url="/static/images/flame-red.svg" />
+          <StyledTooltip content={TooltipContent}>
+            <FlameIcon onClick={handleShowNewUpdates} backgroundColor="yellow.100" url="/static/images/flame-red.svg" />
+          </StyledTooltip>
```

Comment:
Everything looks good to me except I would remove the tool tip for this case where the user haven't see the new updates yet. However this challenges the design, so I think we have to discuss. The reasoning is as follows; 

If the flame icon is appearing in yellow that means the notification drop-down has already appeared before and the user has closed it. So they already know that by clicking on the icon they will see the new updates. To me it doesn't make sense to show the tool-tip in this case, since it just floods the user with too many notifications as if we really want them to click on it. 😄 😄  Also this can also show the tool-tip overlapping with the drop-down notification; 

![Screenshot from 2021-07-02 21-36-22](https://user-images.githubusercontent.com/12435965/124343074-9e68df00-db7d-11eb-97a7-2d2598f58f22.png)

@Betree : My suggestion is simply to remove this for now and only show the tool-tip for the case where there's no new changelog notifications. Let me know if you disagree with this. 🤔 🐨 😄 

```suggestion
          <FlameIcon onClick={handleShowNewUpdates} backgroundColor="yellow.100" url="/static/images/flame-red.svg" />
```

Reply:
Makes sense 👍

Reply:
The tooltip overlapping the yellow message seems to be the real problem. I would personally just hide it when `ChangelogNotificationDropdown` is displayed, but @SudharakaP's solution might be simpler to implement so I'm ok with both approaches.

Reply:
@Betree : Thanks. I've applied the fix to only show the tool-tip for the case where there's no new changelog notifications. Hope it looks better now. Let me know if you see any further issues. 😄